src/convertToObject.js

```diff
@@ -1,12 +1,30 @@
 'use strict';
 
+const toCamelCase = require('./toCamelCase');
+
 /**
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const splitString = sourceString.split(';');
+  const newObject = {};
+
+  splitString.forEach((rule) => {
+    if (rule.trim()) {
+      const [key, value] = rule.split(':');
+
+      const formattedKey = key.includes('-')
+        ? key.trim()
+        : toCamelCase(key.trim());
```

Comment:
The conditional check for the presence of '-' in the key before applying `toCamelCase` seems unnecessary. `toCamelCase` should be able to handle keys with and without hyphens, so you can apply it directly to all keys after trimming.